includes/global-settings-komoju.php

```diff
@@ -0,0 +1,52 @@
+<?php
```

Comment:
This file is mostly just a copy/paste of the old 'settings-komoju.php', but with a slightly different format. `WC_Settings_API` and `WC_Settings_Page` don't quite work the same, despite serving a similar purpose!